src/Symfony/Bundle/FrameworkBundle/Tests/Controller/TemplateControllerTest.php

```diff
@@ -48,10 +49,8 @@ public function testLegacyTwig()
         $twig = $this->getMockBuilder('Twig\Environment')->disableOriginalConstructor()->getMock();
         $twig->expects($this->once())->method('render')->willReturn('bar');
 
-        $container = $this->getMockBuilder('Symfony\Component\DependencyInjection\ContainerInterface')->getMock();
-        $container->expects($this->at(0))->method('has')->willReturn(false);
-        $container->expects($this->at(1))->method('has')->willReturn(true);
-        $container->expects($this->at(2))->method('get')->willReturn($twig);
+        $container = new ContainerBuilder();
```

Comment:
You can use `Container` instead of `ContainerBuilder`. Btw, good catch! We shouldn't mock the container. 😃